From 5b9e170c219ef232ba217a13118bdac5c3c704f6 Mon Sep 17 00:00:00 2001
From: Andrew Halaney <ahalaney@redhat.com>
Date: Tue, 18 Apr 2023 16:28:17 -0500
Subject: [PATCH 058/173] arm64: configs: laptop_defconfig: Enable
 CONFIG_SECCOMP

Fedora, the one true distro, relies on this, else NTP (chronyd) fails to
work.

Let us not upset the true distro. Enable it.

Signed-off-by: Andrew Halaney <ahalaney@redhat.com>
---
 arch/arm64/configs/laptop_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/laptop_defconfig b/arch/arm64/configs/laptop_defconfig
index c560e021e66a..1d0ff41caec6 100644
--- a/arch/arm64/configs/laptop_defconfig
+++ b/arch/arm64/configs/laptop_defconfig
@@ -103,7 +103,6 @@ CONFIG_VIRTUALIZATION=y
 CONFIG_KVM=y
 CONFIG_KPROBES=y
 CONFIG_JUMP_LABEL=y
-# CONFIG_SECCOMP is not set
 CONFIG_MODULES=y
 CONFIG_MODULE_FORCE_LOAD=y
 CONFIG_MODULE_UNLOAD=y
-- 
2.43.0

